src/components/Value/index.tsx

```diff
@@ -0,0 +1,42 @@
+import cn from 'classnames';
+import * as React from 'react';
+
+import { Section } from 'src/components/Section';
+import { Container } from '../Container';
+
+export interface IValue {
+  title: string;
+  summary: string;
+  titleColor: string;
+}
+
+export const Value: React.FunctionComponent<IValue> = ({ title, summary, titleColor }) => {
+  return (
+    <div id={title}>
+      <h2>{title}</h2>
+      <div>{summary}</div>
+    </div>
+  );
+};
+
+export interface IValueSection {
+  values: IValue[];
+}
+export const ValueSection = ({ values }: IValueSection) => {
+  return (
+    <Section id="values" className="pt-16" noPaddingB>
+      <Container title="Our Values" className="border-b px-0 pb-20 pb-16">
+        <div className="container flex flex-no-wrap justify-center md:justify-around md:px-0 sm:flex-wrap">
```

Comment:
I believe the `<Container>` component adds the `container` classname so you shouldn't need to add it here

```suggestion
        <div className="flex flex-no-wrap justify-center md:justify-around md:px-0 sm:flex-wrap">
```